staging/src/k8s.io/kube-scheduler/config/v1beta3/types.go

```diff
@@ -184,6 +184,14 @@ type Plugins struct {
 
 	// PostBind is a list of plugins that should be invoked after a pod is successfully bound.
 	PostBind PluginSet `json:"postBind,omitempty"`
+
+	// MultiPoint is a simplified config section to enable plugins for all valid extension points.
+	// Plugins enabled through MultiPoint will automatically register for every individual extension
+	// point the plugin has implemented. Disabling a plugin through MultiPoint disables that behavior.
```

Comment:
> Disabling a plugin through MultiPoint disables that behavior.

What does "that" refer to?

If you set `multiPoint.disabled = [{name:a}]`, does that disable `a` in all individual points, or just cancel any opinion `multiPoint` has about plugin `a`?

If you set `multiPoint.disabled = [{name:"*"}]`, does that disable all plugins in all individual points, or just cancel any opinion `multiPoint` has about all plugins?

Reply:
It depends on the full config. If you set `multiPoint.disabled = [{name:a}]` this cancels any opinion `multiPoint` has about plugin `a`. So, if `multiPoint` was the only place that plugin was configured, then it disables the plugin entirely. But it can still be enabled through individual extension points (because of the hierarchy, `Specific > MultiPoint > Default`.. if there is no `Specific` then `MultiPoint` is the highest precedence)

For `multiPoint.disabled = [{name:"*"}]`, it's the same behavior just applied to all plugins. This is essentially explicitly saying the config will only rely on Specific extension point configs

Reply:
> For multiPoint.disabled = [{name:"*"}], it's the same behavior just applied to all plugins. This is essentially explicitly saying the config will only rely on Specific extension point configs

I also a concern here. What if `multiPoint.disabled = [{name: "*"}]` is specified along with `multiPoint.enabled = [{name: "A"}]`? Does that mean: (1) plugin A should be enabled, or (2) all plugins disabled. BTW: suppose the regular extension point config is nil.

Reply:
Answered Wei's question above and added cases to cover it https://github.com/kubernetes/kubernetes/pull/105611#discussion_r749410464